frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -240,32 +253,52 @@ const SearchContainerApi: React.FC = () => {
 
   const [applicantFilter, setApplicantFilter] = React.useState<
     ApplicantFilterInput | undefined
-  >(undefined);
+  >({ pools });
 
-  const [{ data: countData, fetching: countFetching }] =
+  // Fetches the number of pool candidates by pool to display on pool cards.
+  const [
+    { data: poolCandidateResultsData, fetching: fetchingPoolCandidateResults },
+  ] = useCountPoolCandidatesByPoolQuery({
+    variables: applicantFilterToQueryArgs(applicantFilter),
+  });
+
+  // Fetches the total number of candidates, since some pool candidates will correspond to the same user.
+  const [{ data: countApplicantsData, fetching: fetchingCandidateCount }] =
     useCountApplicantsQuery({
       variables: applicantFilterToQueryArgs(applicantFilter),
     });
+  const totalCandidateCount = countApplicantsData?.countApplicants || 0;
 
-  const candidateCount = countData?.countApplicants ?? 0;
   const paths = useTalentSearchRoutes();
-  const onSubmit = async () => {
-    return pushToStateThenNavigate(paths.request(), {
-      applicantFilter,
+  const onSubmit = async (candidateCount: number, poolId: string) => {
+    return pushToStateThenNavigate<{
```

Comment:
OK, so this is pushing scope but at the same time it's pretty broken.  :laughing:.  On the admin side, the view request seems to not work anymore.

On the first network request, the pool ID is part of the response that populates the manager information.
![image](https://user-images.githubusercontent.com/8978655/199308377-a6c898d9-fb30-4264-9f0f-ec9ac2456585.png)

But the second network request to populate the list of candidates is missing it so it is returning way too many hits.
![image](https://user-images.githubusercontent.com/8978655/199308583-cdf0aa0e-ec21-4411-a855-8b1b541b06bf.png)

Reply:
Cool, glad it's being tracked elsewhere.  I won't consider that for this PR.